### PR TITLE
Update on Muskit

### PR DESCRIPTION
**Pull request by A-Quarter-Mile**
Hi @ftshijt, here are updates on muskit:

- Add recipe opencpop
- fix bug in `beat` and `label`
- expand xml reader & writer

p.s. Doc for modification to opencpop annotation will be updated later.

### CODE REVIEW

File: egs2/kiritan/svs1/local/extract_from_midi.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""
```
Comment (ftshijt) [inline egs2/kiritan/svs1/local/extract_from_midi.py:1]: empty file?

Comment (A-Quarter-Mile) [reply egs2/kiritan/svs1/local/extract_from_midi.py:1]: This piece of code may help to to extract xml input from midi if needed.
I just add it for kiritan recipe for partners temporarily, if needed.

Comment (ftshijt) [reply egs2/kiritan/svs1/local/extract_from_midi.py:1]: I would suggest to remove this one for the version merge to master. You may add anohter PR to branch `muskit_recipe` instead, for this purpose

Comment (A-Quarter-Mile) [reply egs2/kiritan/svs1/local/extract_from_midi.py]: okay, removed it here.

**Comment by ftshijt** [comment]
Can you also update the `egs2/TEMPLATE/README.md` correspondingly? After that, I will merge thte PR

**Pull request merged**